mcs/class/Mono.Security.Win32/Test/Mono.Security.Cryptography/MD2Test.cs

```diff
@@ -48,7 +48,7 @@ static public void AssertEquals (string msg, byte[] array1, byte[] array2)
 			msg += " -> Expected " + BitConverter.ToString (array1, 0);
 			msg += " is different than " + BitConverter.ToString (array2, 0);
 		}
-		Assertion.Assert (msg, a);
+		Assert.True (a, msg);
```

Comment:
We're using IsTrue instead of True (which I think isn't supported by our version of Nunit)